components/scream/src/physics/p3/p3_rain_sed_impl.hpp

```diff
@@ -152,9 +152,12 @@ ::rain_sedimentation(
       });
     }
     Kokkos::single(
-     Kokkos::PerTeam(team), [&] () {
-      precip_liq_surf += prt_accum * C::INV_RHO_H2O * inv_dt;
-    });
+      Kokkos::PerTeam(team), [&] () {
+        precip_liq_surf += prt_accum * C::INV_RHO_H2O * inv_dt;
+#if defined(SCREAM_FORCE_RUN_DIFF) || defined(SCREAM_FORCE_RUN_DIFF_BFB_UNIT)
```

Comment:
Can you add a comment line here, explaining to puzzled devs that we are just trying to trigger an error, to see if the testing system can catch it?

Reply:
yes